src/components/password.components.js

```diff
@@ -0,0 +1,97 @@
+import $ from 'jquery';
+
+export default class PasswordInput {
+    constructor(queryString, valid = false, disabled = false, required = true) {
+        this.div = $(queryString);
+        this.input = $(`${queryString} input[type="password"]`);
+        this.icon = $(`${queryString} input[type="password"]+i`);
+
+        this.valid = valid;
+        this.disabled = disabled;
+        this.required = required;
```

Comment:
I don't get object value
![image](https://user-images.githubusercontent.com/23269934/163168225-813fc492-550b-4d2b-96cb-c2fa0fe3e0af.png)

Reply:
These are states I want, other information can be private.